src/lib/native.ts

```diff
@@ -757,3 +757,32 @@ export async function writePref(name: string, value: any) {
         write(file, text.replace(substr, `pref("${name}", ${value})`))
     }
 }
+
+/** Obey Mozilla's orders https://github.com/tridactyl/tridactyl/issues/1800 */
+export async function unfixamo() {
+    try {
+        if (localStorage.unfixedamo === "true") {
+            // fixamo already ran for the tridactyl instance in this profile
+            return;
+        }
+        const profile = (await getProfileDir()) + "/";
+        const userjs = await loadPrefs(profile + "user.js");
+        const tridactylPref = "tridactyl.unfixedamo"
+        if (userjs[tridactylPref] === "true") {
+            // fixamo already ran for this firefox profile
+            return;
+        }
+        const prefName = "privacy.resistFingerprinting.block_mozAddonManager"
+        if (userjs[prefName] !== undefined && userjs[prefName] !== "false") {
+            await writePref(prefName, "false");
+            await writePref(tridactylPref, "true");
+        }
+        // Note: we store unfixedamo in localStorage and not in config because
+        //       users might clear their config with :sanitize
+        localStorage.unfixedamo = "true";
+        return;
```

Comment:
We should open a new tab explaining the choices we had to make without the user's consent to avoid being blocked. (If you have time)